pxc-80/Dockerfile

```
@@ -58,29 +58,33 @@ RUN groupadd -g 1001 mysql \
 RUN sed -i '/nodocs/d' /etc/yum.conf || : \
         && yum install -y \
                 yum-utils \
-        && curl -L -o /tmp/percona-xtradb-cluster-server.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-server-8.0.16-27dev.4.2.1.el7.x86_64.rpm \
-        && curl -L -o /tmp/percona-xtradb-cluster-shared.rpm  https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-shared-8.0.16-27dev.4.2.1.el7.x86_64.rpm \
-        && curl -L -o /tmp/percona-xtradb-cluster-client.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-client-8.0.16-27dev.4.2.1.el7.x86_64.rpm \
-        && curl -L -o /tmp/percona-xtradb-cluster-shared-compat.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-shared-compat-8.0.16-27dev.4.2.1.el7.x86_64.rpm \
+        && curl -L -o /tmp/percona-xtradb-cluster-server.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-server-8.0.18-26.4.3.1.el7.x86_64.rpm \
+        && curl -L -o /tmp/percona-xtradb-cluster-shared.rpm  https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-shared-8.0.18-26.4.3.1.el7.x86_64.rpm \
+        && curl -L -o /tmp/percona-xtradb-cluster-client.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-client-8.0.18-26.4.3.1.el7.x86_64.rpm \
+        && curl -L -o /tmp/percona-xtradb-cluster-shared-compat.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-shared-compat-8.0.18-26.4.3.1.el7.x86_64.rpm \
```

```suggestion
        && curl -L -o /tmp/percona-xtradb-cluster-shared-compat.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-shared-compat-${PXC_VERSION}.x86_64.rpm \
```
And change the version in PXC_VERSION variable (line 25). Also please update pxc-80/Dockerfile.k8s file.